src/component/inspect-request.js

```diff
@@ -6,13 +6,32 @@ const {func, object} = React.PropTypes;
 
 export default class InspectRequest extends React.Component {
 
+  _getMaxHeight() {
+    return window.innerHeight * 0.6 - 100;
+  }
+
+  _onResize() {
+    const maxHeight = this._getMaxHeight();
+    this.setState({maxHeight});
+  }
+
   componentWillMount() {
     this.setState({
-      showUrlMapperWindow: false
+      showUrlMapperWindow: false,
+      maxHeight: this._getMaxHeight()
     });
   }
 
+  componentDidMount() {
+    window.addEventListener('resize', this._onResize.bind(this));
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this._onResize.bind(this));
```

Comment:
Does this work? When you run `bind()`, you create a new function, so the old listener will still be there